undyingkingdoms/routes/index/login.py

```diff
@@ -22,3 +22,56 @@ def login():
         else:
             flash("Your email or password was incorrect.")
     return render_template("index/login.html", form=form)
+
+
+class LoginAPI(MethodView):
+    def post(self):
```

Comment:
This code does nothing currently. I was building it as a drop in replacement for the current code. This would allow separating the front and back ends.